ui/app/AppLayouts/Chat/panels/communities/CommunityProfilePopupInviteMessagePanel.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.4
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared 1.0
+import shared.views 1.0
+import shared.status 1.0
+
+ColumnLayout {
+    id: root
+
+    property var pubKeys: ([])
+
+    property var rootStore
+    property var contactsStore
+
+    property alias inviteMessage: messageInput.text
+
+    spacing: Style.current.padding
+
+    QtObject {
+        id: d
+
+        readonly property int maxMsgLength: 140
+        readonly property int msgHeight: 108
+    }
+
+    StatusBaseText {
+        text: qsTr("Invitation Message")
+        font.pixelSize: 13
+    }
+
+    StatusInput {
+        id: messageInput
+        charLimit: d.maxMsgLength
+        text: qsTr("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque interdum rutrum sodales. Nullam mattis fermentum libero, non volutpat.")
```

Comment:
Remove the lorem ipsum.
Do we need a placeholder instead?

Reply:
There is [comment](feat/issue-6382-update-invite-users-to-community-modal) in figma about pre-filled message. for now i'll add placeholder "The message a contact will get with community invitation"